src/components/dashboard/DashboardContainer.js

```diff
@@ -2,7 +2,9 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 
 import DashboardOtherAssets from './DashboardOtherAssets'
+import Balance from '../Balance/Balance'
 
+import milli from '../../images/n-1000.svg'
```

Comment:
I don know but is there a reasob not to import image once in Balance component?

Reply:
it has black one and write one, this is easier to understand as giving as props